_DeploymentAndDistroPackaging/ansible/README.md

```diff
@@ -0,0 +1,33 @@
+# CIAO Deployment with Ansible Playbooks
+
+* Work in Progress
+
+## Prework
+
+ - Deploy SSH key from the host running ansible, to all hosts in the cluster setup.
+ - Install Ansible and his dependencies.
+
+## Environment setup
+
+### Edit the hosts file according to your cluster setup
+```shell
+hosts.yml
+
+[controllers]
+controller.example.com
+
+[networks]
+network.example.com
```

Comment:
Is it the new standard we're going to follow with groups names? we could use ciao_network, ciao_controller, ciao_compute, etc